### DISCUSSION TITLE
3D tiles mesh do not pass through (scroll below)

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#7727] [Q&A]
Hi,

is it possible to set view to not pass through some solid 3D tiles mesh?

E.g. in https://loaders.gl/examples/i3s you can scroll to the 0.

Maybe it's not the best example, but we have som solid mesh data (parts of landscape from drone with elevation) and it would be fine if user cannot scroll below data.

But it's not expected behaviour for all the data. It should be optional per layer?

Thanks

**Comment by Pessimistress** [comment]
You can set `position` in the view state to be a point above ground.

**Reply by radoslavirha** [reply]
@Pessimistress Thanks for your reply. I'm looking for solution to automatically pick correct elevation from 3D tiles and set `position`. 

My use case is similar to local terrain, but data is I3S or cesium. With terrain layer you can also scroll below the terrain (Should be prevented I would say).

Maybe I'll succeed with picking engine, but I'm not sure if it's the best solution.

Thanks
